### PR TITLE
Remote pagination of tx history

### PR DESCRIPTION
**Pull request by isocolsky**

### CODE REVIEW

File: lib/blockchainexplorers/insight.js [hunk 1]
```diff
@@ -84,6 +84,7 @@ Insight.prototype.getTransactions = function(addresses, from, to, cb) {
       addrs: [].concat(addresses).join(',')
     },
   };
+  console.log('*** [insight.js ln87] args:', args); // TODO
```
Comment (matiu) [inline lib/blockchainexplorers/insight.js]: rm logs please.

File: test/integration/server.js [hunk 2]
```diff
@@ -173,7 +173,28 @@ helpers.stubBroadcastFail = function() {
 };
 
 helpers.stubHistory = function(txs) {
-  blockchainExplorer.getTransactions = sinon.stub().callsArgWith(3, null, txs);
+  blockchainExplorer.getTransactions = function(addresses, from, to, cb) {
```
Comment (matiu) [inline test/integration/server.js]: I have seen this code before.

File: test/integration/server.js [hunk 3]
```diff
@@ -260,7 +280,7 @@ helpers.createAddresses = function(server, wallet, main, change, cb) {
 
 var storage, blockchainExplorer;
 
-var useMongo = false;
+var useMongo = true;
```
Comment (matiu) [inline test/integration/server.js]: = false?

File: test/integration/server.js (hunk 2 above)
Comment (isocolsky) [inline test/integration/server.js]: INCEPTION...

**Comment by isocolsky** [comment]
Thanks for the review

**Pull request merged**